tests/unit/test_query_model_transforms.py

```diff
@@ -59,3 +59,79 @@ def test_pick_one_row_per_patient_transform():
 
     transformed = apply_transforms(variables)
     assert transformed == expected
+
+
+def test_adds_one_selected_column_to_sorts():
+    events = SelectTable(
+        "events",
+        TableSchema(i1=Column(int), i2=Column(int)),
+    )
+    by_i1 = Sort(events, SelectColumn(events, "i1"))
+    variables = dict(
+        v=SelectColumn(
+            PickOneRowPerPatient(source=by_i1, position=Position.FIRST), "i2"
```

Comment:
Might help with legibility here and elsewhere if you put a trailing comma in this line, and then black would reformat it so that `"i2"` was on a new line.  This would make it clearer (to me) that `"i2"` is the second argument of the `SelectColumn`.

Reply:
Yeah, I think that's a bit better. I think it's a fools errand to try to make these beautiful without more tooling though.